docassemble/MAVirtualCourt/data/questions/basic-questions.yml

```diff
@@ -29,7 +33,7 @@ objects:
   - plaintiffs: PeopleList
   - plaintiff: Individual
   - other_parties: PeopleList
-  - courts: DAList.using(object_type=MACourt, there_are_any=True, there_is_another=False) # Normally let's only have one court...
+  - courts: DAList.using(auto_gather=False, gathered=True, complete_attribute='name') # Normally let's only have one court...
```

Comment:
Is this change necessary for this PR?

Reply:
> 
> 
> > Is there a way we can make that less confusing?
> 
> Other options:
> 
>     1. A link that sends an email to one of us.
> 
>     2. A link to a google form they can turn in with the url pre-populated for them
> 
>     3. A 'copy this' type of interaction (where clicking on the link copies the link to their clipboard - no, I don't know how to do this yet) with instructions to share the link in Slack, with a certain email address, on a google form, whatever.

Grea

> Other options:
> 
>     1. A link that sends an email to one of us.

These are great ideas, but outside of scope of this PR. I think this is just a stopgap until someone creates the form/process described above, which I think is really ideal. I'd prefer that someone not to be me.

Reply:
Sure, I hear that. I'll make a commit on this branch to add instructions at least.